### PR TITLE
fix building bug & add cd docs

### PR DESCRIPTION
**Pull request by motahharm**
Hi there.

there was an bug when i run `hugo server -D`:
```
Error: add site dependencies: load resources: loading templates: "/mnt/d/MainProject/the-secret-bit/themes/cupper-hugo-theme/layouts/partials/head.html:14:1": parse failed: template: partials/head.html:14: unclosed action
```

I fix this

and I add cd command docs but it is draft

### CODE REVIEW

**Comment by shabane** [comment]
hi =)
im too happy that you helped us to improve the book.

this error is due to that you need the `cupper` theme in the `theme` directory
you have two choice for this:
1. download the [theme](https://github.com/zwbetz-gh/cupper-hugo-theme) into the directory manually 
2. or you can clone the repo with `--recurse-submodules`

the problem with the **draft** is that you should mark **draft** as _false_ in the cp.md file.
to do this edit line 4 of your file.
**thanks** you again. i will issue some problems then we will merge your commit.

File: content/posts/cd/cd.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+<div dir='rtl'>
+
+[!["cd command usage"](cd.jpg)](cd.jpg)
+
+### فهرست
+
+> - [مقدمه](#مقدمه)
+> - [رفتن به یک دایرکتوری](#رفتن-به-یک-دایرکتوری)
+> - [رفتن به پوشه های موجود](#رفتن-به-پوشه-های-موجود)
```
Comment (shabane) [inline content/posts/cd/cd.md:15]: اینجا بهتره بگی رفتن به دایرکتوری های تو در تو

File: content/posts/cd/cd.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+```
+</div>
+
+!["an example for cd"](cd_ex1.jpg)
+
+---
+
+### رفتن به پوشه های موجود
+
+اگه در پوشه ای قرار دارید که در آن پوشه های موجود دیگری باشند می‌توانید با دستور
```
Comment (shabane) [inline content/posts/cd/cd.md:58]: پوشه های دیگری موجود باشند

File: content/posts/cd/cd.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+<div dir='ltr'>
+
+```
+$ cd <DIR_NAME>
```
Comment (shabane) [inline content/posts/cd/cd.md:62]: `cd DIR1/DIR2/DIR3/`

File: content/posts/cd/cd.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+```
+
+</div>
+رفتن به پوشه مورد نظر خود را انجام دهید.
+
+همچنین در هر دایرکتوری برای برگشت به دایرکتوری پدر خود با دستور
```
Comment (shabane) [inline content/posts/cd/cd.md:68]: بجای پدر بنویسیم قبلی بهتره

Comment (shabane) [reply content/posts/cd/cd.md:68]: و اینو اضافه کن که همیشه `..` به پوشه قبلی اشاره می‌کنه.

**Review by shabane — CHANGES_REQUESTED** [review]

**Comment by motahharm** [comment]
Ok @shabane 

i fix them.

File: content/posts/cd/cd.md (hunk 2 above)
Comment (motahharm) [reply content/posts/cd/cd.md]: اشتباه تایپی شده😬😆

**Comment by shabane** [comment]
thanks very much i appreciate you

**Pull request merged**